src/explore-education-statistics-common/src/components/Accordion.tsx

```diff
@@ -177,7 +180,11 @@ const Accordion = ({
             }}
           >
             {isAllOpen ? 'Close all ' : 'Open all '}
-            <span className="govuk-visually-hidden">sections</span>
+            {!visuallyHiddenText ? (
+              <VisuallyHidden>sections</VisuallyHidden>
+            ) : (
+              <VisuallyHidden>{visuallyHiddenText}</VisuallyHidden>
+            )}
```

Comment:
This could be simplfied a bit, something like this:
```
<VisuallyHidden>{visuallyHiddenText ? ` ${visuallyHiddenText}` : ' sections'}</VisuallyHidden>
```

Also, don't forget that we need the spaces before the hidden text so that's it's read correctly by screen readers.